Sources/ApolloCodegenLib/Templates/SchemaConfigurationTemplate.swift

```diff
@@ -1,27 +1,29 @@
 import Foundation
 
 /// Renders the Cache Key Resolution extension for a generated schema.
-struct SchemaCacheKeyResolutionTemplate: TemplateRenderer {
+struct SchemaConfigurationTemplate: TemplateRenderer {
 
   /// Source IR schema.
   let schema: IR.Schema
   /// Shared codegen configuration
   let config: ApolloCodegen.ConfigurationContext
 
-  let target: TemplateTarget = .schemaFile(type: .cacheKeyResolutionExtension)
+  let target: TemplateTarget = .schemaFile(type: .schemaConfiguration)
```

Comment:
This and the renaming of the `.schema` case too are much better names, I'm glad we did this change.